lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
I was thinking separately, something like:

```
default_settings :foo => ["value", :required], :bar => "another_value 
```

although I would rename default_settings to settings.